git/commit-message-template.md

```diff
@@ -0,0 +1,47 @@
+# Today I Learned...How to setup a commit message template
+
+This is another that I've actually been using for a while, but feels worth adding here.
+
+If you've used Git even a little, you probably recognize this message:
+
+```sh
+# Please enter the commit message for your changes.
+# (Comment lines starting with '#' will not be included)
+# Explicit paths specified without -i nor -o; assuming --only paths...
+# On branch master
+# Changes to be committed:
+#   (use "git reset HEAD <file>..." to unstage)
+#
+#       modified:   banana.txt
+#
+```
+
+What many people don't realize, is that you can actually customize that message. Doing so allows you to not only have a nice guide for your
```

Comment:
I think that comma is misplaced, but I don't know where it's supposed to go, so I'm improvising a suggestion:
```diff
- What many people don't realize, is that you can actually customize that message.
+ You can actually customize that message!
```

Reply:
I just want to clarify-- that default message is supposed to show up every time, correct? I followed your instructions -- my commit message shows up before the default, but it doesn't replace the default like I originally thought.